Twoliter.toml

```diff
@@ -1,15 +1,21 @@
 schema-version = 1
-release-version = "1.21.0"
+release-version = "1.21.1"
 
 [vendor.bottlerocket]
 registry = "public.ecr.aws/bottlerocket"
 
+# ======= REVERT BEFORE FINAL PR =================
+[vendor.monirul]
+registry = "397850904080.dkr.ecr.us-west-2.amazonaws.com"
```

Comment:
Please don't use non-Bottlerocket vendors in a PR unless it's marked as draft. Too easy for it to be merged by accident.

Reply:
I marked the PR as draft.